cumulusci/tasks/robotframework/tests/test_robotframework.py

```diff
@@ -292,6 +292,44 @@ def test_sources_not_found(self, mock_robot_run):
             task()
 
 
+@mock.patch("cumulusci.tasks.robotframework.robotframework.robot_run")
+def test_outputdir_return_value(mock_run, tmpdir):
+    """Ensure that the task properly sets the outputdir return value"""
+    universal_config = UniversalConfig()
+    project_config = BaseProjectConfig(
+        universal_config,
+        {
+            "sources": {
+                "test1": {"path": "dummy1"},
+                "test2": {"path": "dummy2"},
+            }
+        },
+    )
+    # get_namespace returns a config. The only part of the config
+    # that the code uses is the repo_root property, so we don't need
+    # a full blown config.
+    project_config.get_namespace = mock.Mock(
+        side_effect=lambda source: mock.Mock(
+            repo_root=project_config.sources[source]["path"]
+        )
+    )
```

Comment:
Why not just leave `sources` out of the task options and then you don't need to mock this?

Reply:
This is definitely not needed. I'll remove it.